common/web/types/src/main.ts

```diff
@@ -31,3 +31,5 @@ export { VariableParser, MarkerParser } from './ldml-keyboard/pattern-parser.js'
 export { ElementString } from './kmx/kmx-plus/element-string.js';
 
 export { USVString, CasingForm, CasingFunction, TextWithProbability, LexiconTraversal, LexicalModel, LexicalModelPunctuation, Transform, Suggestion, Reversion, Keep, SuggestionTag, Context, Distribution, Outcome, WithOutcome, ProbabilityMass, Configuration, Capabilities, WordBreakingFunction, Span } from './lexical-model-types.js';
+
+export { ComplexKeyboardStore, EncodedVisualKeyboard, LayoutSpec, KeyboardObject } from './keyboard-object.js';
```

Comment:
```suggestion
export * as KeymanWebKeyboard from './keyboard-object.js';
```

I'd still like to try and export this as a grouped type, given `KeyboardObject` etc are really KMW-specific objects.

Reply:
ah, ok. I meant to do it as part of #12516, but you're right, it makes more sense to do it now. Done.